lib/versions.js

```diff
@@ -6,6 +6,21 @@ import { HttpClient } from "@actions/http-client";
 export async function determineVersion(version, { repo, nightliesRepo }) {
   version = version?.trim();
 
+  const toolVersionsPathInput = core.getInput("tool-versions");
```

Comment:
Could you please add a check that if user provides **both** `scarb-version` and `tool-versions` inputs, the action will yell at the user?